Changelogs for Digital Twin Manager UI 1024.0.0 (#4137)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1024-0-0-fix-for-assets-list-appearing-empty-despite-available-assets.md

---
date: ""

> title: "Asset list no longer empty despite available assets

Reviewer: ```suggestion
title: "Asset list no longer empty despite available assets"
```

> title: "Fix for Assets list appearing empty despite available assets."

Reviewer 2: ```suggestion
title: "Asset list no longer empty despite available assets
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2627"
version: "1024.0.0"
---

> Previously, the assets grid in Assets page would occasionally appear empty even when assets were available; this is now resolved

Reviewer 2: ```suggestion
Previously, the asset list in the **Assets** page occasionally appeared empty even when assets were available. This issue has been fixed, and available assets are now properly displayed.
```

---
date: ""
title: "Fix for Assets list appearing empty despite available assets."
product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2627"
version: "1024.0.0"
---
In the Assets page, resolved an issue that occasionally caused the

> assets table to appear empty, despite assets being present.

Reviewer: ```suggestion
Previously, the assets grid in Assets page would occasionally appear empty even when assets were available; this is now resolved
```
